src/UNL/UCBCN/Manager/Auth.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace UNL\UCBCN\Manager;
+use SimpleCAS;
+use UNL\UCBCN\User as User;
+
+class Auth {
+
+	protected $options = array();
+    
+    public static $directory_url = 'http://directory.unl.edu/';
+
+    /**
+     * Authenticate the user
+     */
+    public function authenticate() {
+        $client = $this->getClient();
+        
+        $client->forceAuthentication();
+        if (!$client->isAuthenticated()) {
+            throw new RuntimeException('Unable to authenticate', 403);
+        }
+        
+        $user = $this->getUser($client->getUsername());
+        
+        print_r($user);
```

Comment:
@tlemburg This is very early in the development, so I'm not really worried about most of these things yet.  I just want to make sure that you are aware of this debug statement, some unnecessary commented code and formatting issues in this file.